lib/tasks/bulkrax_tasks.rake

```diff
@@ -1,6 +1,30 @@
 # frozen_string_literal: true
 
-# desc "Explaining what the task does"
-# task :bulkrax do
-#   # Task goes here
-# end
+namespace :bulkrax do
```

Comment:
this rake task needs to be run on all projects that update to this version!!

the exporter paths have all changed, so existing exported files can't be reached anymore.